regexes/client/browsers.yml

```
@@ -2420,7 +2420,7 @@
   version: '$1'
 
 # MAUI Wap Browser
-- regex: 'WAP Browser/MAUI|(?:\w+)?Maui Wap Browser|MAUI[- ]Browser'
+- regex: 'WAP Browser/MAUI|(?:.*_)?Maui Wap Browser|MAUI[- ]Browser'
```

Doesn't that actually change the regex? Wouldn't that change be needed to keep that same behaviour?
```suggestion
- regex: 'WAP Browser/MAUI|(?:\w*)Maui Wap Browser|MAUI[- ]Browser'
```